modules/wishlist-magento/services/useAddToWishlist.ts

```diff
@@ -15,12 +15,10 @@ export default () => {
     const items = [...wishlistStore.items, { id: product.id, product }]
     const productIds = items.map((item) => item.product.sku)
 
-    if (customer.isLoggedIn) {
-      await addToWishlistRepository(product)
-    } else {
+    if (!customer.isLoggedIn) {
       cookies.set(WISHLIST_COOKIES_NAME, productIds, { path: '/' })
     }
-
     wishlistStore.$patch({ items })
+    return await addToWishlistRepository(product)
```

Comment:
You cant call the addToWishlistRepository when user is not logged in. Why this change to the service?